### PR TITLE
[FEATURE] Implemented 58-BLoc structure

### PR DESCRIPTION
**Pull request by Shashank250705**

### CODE REVIEW

**Comment by Shashank250705** [comment]
> Not navigating from home tab structure to get ride screen

i will make the required changes

File: lib/screens/home_screen_tab_structure.dart [hunk 1]
```diff
@@ -32,170 +38,169 @@ class _HomeScreenState extends State<HomeScreen> {
...
+            ),
+          ],
+        ),
+      ),
     );
   }
 }
 
-class ScheduleTile extends StatefulWidget {
+class ScheduleTile extends StatelessWidget {
```
Comment (ushamsharath) [inline lib/screens/home_screen_tab_structure.dart:183]: convert into to StatefulWidget

File: lib/screens/home_screen_tab_structure.dart [hunk 2]
```diff
@@ -32,170 +38,169 @@ class _HomeScreenState extends State<HomeScreen> {
...
     return RideDetailsTile(
-      values: widget.values,
+      values: values,
       path: 'two',
     );
   }
 }
 
-class RequestTile extends StatefulWidget {
+class RequestTile extends StatelessWidget {
```
Comment (ushamsharath) [inline lib/screens/home_screen_tab_structure.dart:196]: this one also

File: lib/main.dart [hunk 3]
```diff
@@ -14,17 +16,22 @@ Future<void> main() async {
...
+      child: MaterialApp(
+        title: 'RideBhaiya',
+        theme: ThemeData(
+          colorScheme: ColorScheme.fromSeed(seedColor: const Color(0xFF49B6F3)),
+          useMaterial3: true,
+        ),
+        home: const SplashScreen(),
       ),
-      home:
-          const SplashScreen(), // when you change this class name in local, you need to revert it back to SplashScreen() before pushing to GitHub
```
[the remark this answers is not in the document]
Comment (maheshalyana) [reply lib/main.dart:27]: No issue for this one

File: lib/main.dart [hunk 4]
```diff
@@ -14,17 +16,22 @@ Future<void> main() async {
 class MyApp extends StatelessWidget {
   const MyApp({super.key});
 
-  // This widget is the root of your application.
```
[the remark this answers is not in the document]
Comment (maheshalyana) [reply lib/main.dart:17]: no issue for this

**Review by ushamsharath — APPROVED** [review]

**Review by tanishq-2208 — APPROVED** [review]

**Review by Abhi-0987 — APPROVED** [review]
https://github.com/NewTab-Technnologies/Ride-Bhaiya/assets/124353488/d0aa0623-1212-4ed1-8c25-2c739dd2c6fc
image

The flow id correct and the test cases passed. Moving it to acceptance.

**Pull request merged**